File: DSA/AadrikaBehl_590012053/day6q2.c
#include <stdio.h>
#include <stdbool.h>

bool isStrictlyIncreasing(int nums[], int size) {
    for (int i = 1; i < size; i++) {
        if (nums[i] <= nums[i - 1]) {
            return false;
        }
    }
    return true;
}

int main() {
    int n;
    printf("Enter the number of elements in the array: ");
    scanf("%d", &n);

    int nums[n];
    printf("Enter %d integers:\n", n);
    for (int i = 0; i < n; i++) {
        scanf("%d", &nums[i]);
    }

    if (isStrictlyIncreasing(nums, n)) {
        printf("true\n");
    } else {
        printf("false\n");
    }

    return 0;
}

File: DSA/NarindVerma_500121898/Day6Q2.py
def isStrictlyIncreasing(nums):
    for i in range(1, len(nums)):
        if nums[i] <= nums[i - 1]:
            return False
    return True

# Examples
print(isStrictlyIncreasing([1, 2, 3, 4]))     # Output: True
print(isStrictlyIncreasing([1, 2, 2, 3]))     # Output: False
print(isStrictlyIncreasing([5, 4, 3]))        # Output: False
